server/pulp/server/webservices/views/consumers.py

```diff
@@ -24,6 +28,80 @@
                                                 json_body_allow_empty)
 
 
+def add_link(consumer):
+    """
+    Add link to the consumer object.
+
+    :param consumer: consumer object
+    :type consumer: dict
+
+    :return: link containing the href
+    :rtype: dict
+    """
+
+    link = {'_href': reverse('consumer_resource',
+            kwargs={'consumer_id': consumer['id']})}
+    consumer.update(link)
+    return link
+
+
+def add_link_profile(consumer):
+    """
+    Add link to the consumer profile object.
+
+    :param consumer: consumer profile object
```

Comment:
It seems confusing to call the consumer profile consumer instead of consumer_profile.